cfme/test_requirements.py

```diff
@@ -420,3 +420,9 @@ def test_quota_alert():
     description='Support of VMWare in MIQ/CFME',
     assignee_id='kkulkarn'
 )
+
+multi_region = pytest.mark.requirement(
```

Comment:
This file is _roughly_ alphabetical, mind putting this in the middle of the list? (`multi-tenancy` is out of place too if you want to move it...)